rust/core-lib/src/styles.rs

```diff
@@ -32,22 +34,23 @@ pub struct Style {
 impl Style {
     // construct the params for a def_style request
     pub fn to_json(&self, id: usize) -> Value {
-        let mut builder = ObjectBuilder::new()
-            .insert("id", id)
-            .insert("fg_color", self.fg);
-        if (self.bg >> 24) > 0 {
-            builder = builder.insert("bg_color", self.bg);
-        }
-        if self.weight != 400 {
-            builder = builder.insert("weight", self.weight);
-        }
-        if self.underline {
-            builder = builder.insert("underline", self.underline);
-        }
-        if self.italic {
-            builder = builder.insert("italic", self.italic);
-        }
-        builder.build()
+        let mut json = json!({
+            "id": id,
+            "fg_color": self.fg,
+        });
+            if (self.bg >> 24) > 0 {
```

Comment:
The indentation looks weird to me. If you want these indented, maybe add a {} block?